galgebra/ga.py

```diff
@@ -32,52 +33,61 @@ def is_bases_product(w):
     return len(nc) == 2 or len(nc) == 1 and nc[0].is_Pow and nc[0].exp == 2
 
 
-class auto_update_dict(dict):
```

Comment:
This has been here but unused since [three universes](https://github.com/pygae/galgebra/blob/7c3fd55df034e20ca2b268b9f05ca9a192713597/ga.py#L27) before [the beginning of time](https://github.com/pygae/galgebra/commit/fab9012af52b2c9cee378ea0f923394dd3383d0b#commitcomment-36053978)

Reply:
I suspect @abrombo intended to use this in exactly this way :)